tools/acrntrace/README.rst

```diff
@@ -12,6 +12,67 @@ A ``scripts`` directory includes scripts to analyze the trace data.
 Usage
 *****
 
+The ``acrntrace`` tool runs on the Service OS (SOS) to capture trace data and
+output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
+
+Options:
+
+-h			print this message
+-t period		specify polling interval in milliseconds [1-999]
+-c			clear the buffered old data
+
+The ``acrntrace_format.py`` is a offline tool for parsing trace data (as output
+by acrntrace) to human-readable formats based on given format.
+
+Here's an explanation of the tool's parameters:
+
+.. code-block:: none
+
+   acrntrace_format.py [options] [formats] [trace_data]
+
+Options:
+
+-h    print this message
+
+- A *formats* file specifies the rules to reformat the *trace_data* collected by
+  ``acrntrace`` into a human-readable text form.  The rules in this file are of
+  the form::
+
+    event_id  text_format_string
+
+  The text_format_string may include format specifiers, such as
+  ``%(cpu)d``, ``%(tsc)d``, ``%(event)d``, ``%(1)d``, and ``%(2)d``.
+  The 'd' format specifier outputs in decimal, alternatively 'x' will
+  output in hexadecimal and 'o' will output in octal.
+
+  These respectively correspond to the CPU number (cpu), timestamp
+  counter (tsc), event ID (event) and the data logged in the trace file.
+  There can be only one such rule for each type of event.
+
+  A pre-defined formats file is available in tools/acrntrace/scripts/formats
+  in acrn source tree
+
+The ``acrnalyze.py`` is a offline tool to analyze trace data (as output by
+acrntrace) based on given analyser, such as ``vm_exit`` or ``irq``.
+
+Options:
+
+-h                               print this message
```

Comment:
Sorry, I had some errors in this option list layout (can't use square brackets):

    -h                                print this message
    -i, --ifile=string                input file name
    -o, --ofile=string                output filename
    -f, --frequency=unsigned_int      TSC frequency in MHz
    --vm_exit                         generate a vm_exit report
    --irq                             generate an IRQ-related report

Reply:
Thanks @dbkinder. Will fix it.